### PR TITLE
Page functionality for create job post.

### PR DESCRIPTION
**Pull request by Tsaikkari**
createJobPost

### CODE REVIEW

File: src/components/JobPostForm/index.tsx [hunk 1]
```diff
@@ -26,7 +31,13 @@ const JobPostForm = ({ header }: HeaderProps) => {
     day: 31,
   }
 
+  const user = useSelector((state: AppState) => state.user)
+  const { loading, error } = user
+
+  const dispatch = useDispatch()
+
   const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const startingDate = startingAt
```
Comment (Wyfy0107) [inline src/components/JobPostForm/index.tsx:40]: Why need to assign again. You can use it directly

File: src/components/Tabs/index.tsx [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import React from 'react'
 import { Tab, Row, Col, Nav } from 'react-bootstrap'
-import LogOut from '../LogOut'
```
Comment (Wyfy0107) [inline src/components/Tabs/index.tsx:3]: Don't we need this ?

File: src/components/JobPostForm/index.tsx (hunk 1 above)
Comment (Tsaikkari) [reply src/components/JobPostForm/index.tsx:40]: Okey.

File: src/components/Tabs/index.tsx (hunk 2 above)
Comment (Tsaikkari) [reply src/components/Tabs/index.tsx:3]: I included Dilip's navbar in this component and it includes the LogOut component. It looks better that way.

Comment (Tsaikkari) [reply src/components/Tabs/index.tsx:3]: I'll close this pr because I just recognized I've been working on this same branch now on redux.

**Pull request closed without merging**